### PR TITLE
Create Consensus transactions list components

### PR DESCRIPTION
**Pull request by buberdds**

### CODE REVIEW

File: src/app/components/Transactions/ConsensusTransactions.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+              key: 'to',
+            },
+            {
+              align: TableCellAlign.Right,
+              content: <RoundedBalance value={transaction.fee} ticker={transaction.ticker} />,
+              key: 'fee_amount',
+            },
+            {
+              align: TableCellAlign.Right,
+              // TODO: show RoundedBalance when API returns amount prop
```
Comment (csillag) [inline src/app/components/Transactions/ConsensusTransactions.tsx:128]: So, both the amount and the receiver of the TX is missing from the API? 😞

**Review by csillag — APPROVED** [review]
Reviewed, LGTM.

**Comment by csillag** [comment]
I would consider using "??" instead of "-" where we have missing data (ie. data that we know exists but is not yet exposed by the API), since "-" could be misinterpreted as meaning "none". What do you think, @donouwens ?

File: src/app/components/Transactions/ConsensusTransactions.tsx (hunk 1 above)
Comment (buberdds) [reply src/app/components/Transactions/ConsensusTransactions.tsx:128]: There is backend ticket to fix that https://app.clickup.com/t/869353k3r

Comment (buberdds) [reply src/app/components/Transactions/ConsensusTransactions.tsx:128]: note for myself: update API layer when props are added (fromBaseUnits) etc

**Comment by lukaw3d** [comment]
I like '-'

but here's how some other options would look like:
image

**Comment by csillag** [comment]
My order of preference:

`-`  < `n/a` < image < `...` < `?` = `??`

**Pull request merged**